pgxn/neon/pagestore_smgr.c

```diff
@@ -239,7 +240,9 @@ typedef struct PrefetchState
 								 * also unused */
 
 	/* the buffers */
-	prfh_hash  *prf_hash;
+	prfh_hash	*prf_hash;
+	int			max_shard_no;
+	uint8		shard_bitmap[(MAX_SHARDS + 7)/8];
```

Comment:
This 'shard_bitmap' desperately needs a comment to explain what it is. And SET/GET helper functions for setting/clearing bits.

Reply:
Done